### PR TITLE
Fix build for trailblazer 2.1

### PR DESCRIPTION
**Pull request by seuros**

### CODE REVIEW

File: lib/trailblazer/rails/controller.rb [hunk 1]
```diff
@@ -1,15 +1,10 @@
 module Trailblazer::Rails
   module Controller
-    def run(operation, params=self.params, *dependencies)
-      result = operation.(
-        _run_params(params),
-        *_run_runtime_options(*dependencies)
-      )
+    def run(operation, params=self.params.clone.permit!, *dependencies)
```
[the remark this answers is not in the document]
Comment (seuros) [reply lib/trailblazer/rails/controller.rb]: permitI is part of https://github.com/rails/strong_parameters not rails 5.1

**Comment by seuros** [comment]
I will add the tests for rails 3.2, 4.2 asap

**Comment by apotonick** [comment]
Still a bit nervous about the `permit!`, we shouldn't assume everyone uses Reform with TRB.

File: lib/trailblazer/rails/controller.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (seuros) [reply lib/trailblazer/rails/controller.rb]: You are right, let remove this hack.

**Pull request merged**